### ISSUE TITLE
Mount 1131 Successful on iPad Pro 9.7

### ISSUE THREAD

**Issue by Co2333** [ethereus/Sn0wf4ll#2]
https://github.com/Co2333/PoCJB1131/blob/master/README.md

I use my own repo for easy manage. I will delete it you like.

**Comment by Co2333** [comment]
Yes.  As it is super unstable and may cause problem with future jailbreak if you don’t have shsh2.

**Comment by DennYzzx** [comment]
Did you try sn0wf4ll jailbreak with cydia?

**Comment by DennYzzx** [comment]
And do you have twitter i would like to follow you

**Comment by Co2333** [comment]
No, and never.and I don't want to give the twitter :P
